style.css

```diff
@@ -112,19 +121,52 @@ button {
   background-color: rgb(163, 20, 163);
 }
 
+
 #game-over {
-  justify-content: center;
-  align-items: center;
-  width: 100%;
   position: absolute;
-  top: 200px;
+  top: 45%;
```

Comment:
With these changes 'game over' isn't centered on smaller screens. Could you please fix this? 
![image](https://user-images.githubusercontent.com/67926235/96000526-a9d57800-0e3f-11eb-85d1-8af912f927d2.png)

Reply:
sure